po/es.po

```diff
@@ -0,0 +1,155 @@
+# Docklike-plugin i18n file
+# Copyright (C) 2020 Nicolas Szabo
+# This file is distributed under the same license as the Docklike package.
+# Nicolas Szabo <nszabo@vivaldi.net>, 2020.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Docklike 0.9\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-12-05 19:00-0300\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Santiago Soler <santiago.r.soler@gmail.com>\n"
+"Language-Team: ES\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: ../src/Plugin.cpp:78
+msgid "Contributors:"
+msgstr "Contribuidores :"
+
+#: ../src/Plugin.cpp:89
+msgid "A modern, minimalist taskbar for XFCE."
+msgstr "Una moderna y minimalista barra de tareas para XFCE"
+
+#: ../src/Wnck.cpp:185
+msgid "Launch another"
+msgstr "Nueva instancia"
+
+#: ../src/Wnck.cpp:185
+msgid "Launch"
+msgstr "Lanzar"
+
+#: ../src/Wnck.cpp:200
+msgid "Unpin"
+msgstr "Desanclar"
+
+#: ../src/Wnck.cpp:200
+msgid "Pin"
+msgstr "Anclar"
+
+#: ../src/Wnck.cpp:223
+msgid "Close All"
+msgstr "Cerrar Todas"
+
+#: ../src/SettingsDialog.cpp:21
+#, c-format
+msgid "<b>Only the first %u hotkeys(s) are enabled.</b>\n"
+msgstr "<b>Solo los %u primeros atajos de teclado están activados.</b>\n"
+
+#: ../src/SettingsDialog.cpp:25
+#, c-format
+msgid ""
+"The &lt;SUPER&gt;+%u combination seems already in use by another process.\n"
+"Check your Xfce settings."
+msgstr ""
+"La combinación &lt;SUPER&gt;+%u está siendo utilizada por otro proceso.\n"
+"Verifique sus configuraciones de Xfce."
+
+#. namespace SettingsDialog
+#: ../src/docklike.desktop.in.h:1
+msgid "A modern, minimalist taskbar for XFCE"
+msgstr "Una moderna y minimalista barra de tareas para XFCE"
+
+#: ../src/_dialogs.xml.h:1
+msgid "_Close"
+msgstr "Cerrar"
+
+#: ../src/_dialogs.xml.h:2
+msgid "Behavior"
+msgstr "Comportamiento"
+
+#: ../src/_dialogs.xml.h:3
+msgid "Don't show list for a single window"
+msgstr "No mostrar la lista para una única ventana"
+
+#: ../src/_dialogs.xml.h:4
+msgid "Show only applications from the current workspace"
+msgstr "Mostrar solo aplicaciones del escritorio actual"
+
+#: ../src/_dialogs.xml.h:5
+msgid "Show only applications from the current display"
+msgstr "Mostrar solo aplicaciones de la pantalla actual"
+
+#: ../src/_dialogs.xml.h:6
+#, fuzzy
+msgid "Appearance"
+msgstr "Apariencia"
+
+#: ../src/_dialogs.xml.h:7
+msgid "Indicator style :"
+msgstr "Estilo del indicador :"
+
+#: ../src/_dialogs.xml.h:8
+msgid "Bars"
+msgstr "Barras"
+
+#: ../src/_dialogs.xml.h:9
+msgid "Dots"
+msgstr "Puntos"
+
+#: ../src/_dialogs.xml.h:10
+msgid "Rectangles"
+msgstr "Rectángulos"
+
+#: ../src/_dialogs.xml.h:11
+#, fuzzy
+msgid "Indicator orientation :"
+msgstr "Posición del indicador :"
```

Comment:
I translated this to something like `Indicator position` rather than `Indicator orientation`. I think that `position` sounds better, even for English. Maybe that's something that could be changed on a different PR.

Let me know what do you think.

Reply:
Haha, this is debatable :)